src/backend/egl/ffi.rs

```diff
@@ -356,4 +356,9 @@ pub mod egl {
     // Accepted in the <attribute> parameter of eglQueryWaylandBufferWL:
     pub const EGL_TEXTURE_FORMAT: i32 = 0x3080;
     pub const WAYLAND_Y_INVERTED_WL: i32 = 0x31DB;
+
+    pub const PLATFORM_ANGLE_ANGLE: u32 = 0x3202;
+    pub const PLATFORM_ANGLE_TYPE_ANGLE: i32 = 0x3203;
+    pub const PLATFORM_ANGLE_NATIVE_PLATFORM_TYPE_ANGLE: i32 = 0x348F;
+    pub const PLATFORM_ANGLE_TYPE_VULKAN_ANGLE: i32 = 0x3450;
```

Comment:
Of course ANGLE doesn't seem to have an extension to define these?

Reply:
gl-generator is effectively abandoned so these might not be in the registry. It's a non trivial task to update gl-generator either

Reply:
I tried to update this in gl-generator (https://github.com/brendanzab/gl-rs/pull/544), but the master branch seems to run into other issues in Smithay currently. So just defining here seems to be the best option currently.

Reply:
Have you tried some newer generators like the one used by glow?

Reply:
phosphorus only works with gl.xml, egl.xml support had a pull request but that was abandoned

Reply:
Well, certainly unfortunate, but I guess this works.